artifacts/CAMARA_common.yaml

```
@@ -5,8 +5,8 @@ info:
   license:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
-  version: 0.5.0
-  x-camara-commonalities: 0.4.0
+  version: 0.6.0-alpha

```

Version numbers should usually be left as `wip` until just before release (for otherwise different versions of the file can have the same release version number within the github history).

If you want to put the alpha version number in now, it should be 0.6.0-alpha.1. See [here](https://lf-camaraproject.atlassian.net/wiki/spaces/CAM/pages/14557919/API+versioning#API-version-extensions).
```suggestion
  version: wip
```